code/modules/reagents/chemistry_reactions/medical.dm

```diff
@@ -50,6 +50,13 @@
 	required_reagents = list("sugar" = 1, "phosphorus" = 1, "sulfur" = 1,)
 	result_amount = 3
 
+/datum/chemical_reaction/hypervene
+	name = "Hypervene"
+	id = "hypervene"
+	result = "hypervene"
+	required_reagents = list("arithrazine" = 1, "dylovene" = 1, "ethylredoxrazine" = 1,)
```

Comment:
stray comma.

Reply:
Stray commas don't actually impact list contents

Reply:
they impact my feelings @JamesDenholm

Reply:
Man if you wanna talk about feelings you should have seen me reeeing it up on discord